### DISCUSSION TITLE
Question to clarify the naming convention for data transfer to Azure Blob Storage

### DISCUSSION THREAD

**Discussion by heschmidt04** [chorus-ai/StandardsModule#83] [Results Sharing Questions] [answered]
This came up when I worked with Seattle to test data transfer today, so I am asking it here and cc:ing Vijay so he's in the loop of communication. 

One image under [Target Folder Structure](https://chorus-ai.github.io/data_acq_SOP/docs/Data-Uploading/Data_uploading/#target-folder-structure) shows laying out the directory structure as ~/mgh/###/ with Image, OMOP, and Waveforms directories underneath - as person id is just a number 

The example under the heading Folder Architecture Details using Person123 shows laying out the directory structure as ~/mgh/person###/ with Image, OMOP, and Waveforms directories underneath - as person is the prefix to the person id number

I tested my az batch upload using this directory structure layout below and want to know which is the winning method, so I don't confuse anyone going forward. 

~/mgh
└── person711
    ├── Images
    ├── OMOP
    └── Waveforms

5 directories, 2585 files

**Comment by heschmidt04** [comment]
Del Bold followed up with the following: 

"Person" string is not prefix and it should be whatever your person_id for OMOP. 
The person _id should be same across all data modality for that person's folder. 

That means the directory structure will be the following - where ### is the number for the person_id 

~/mgh
└── ###
    ├── Images
    ├── OMOP
    └── Waveforms

Hope that helps other folks. 
Best regards,
Heidi

**Comment by del42** [comment, marked as answer]
@heschmidt04  This was discussed here: 
[https://github.com/chorus-ai/data_acq_SOP/discussions/14#discussioncomment-7721999](https://github.com/chorus-ai/data_acq_SOP/discussions/14#discussioncomment-7721999)
